src/platform/packages/shared/presentation/presentation_publishing/interfaces/fetch/applies_filters.ts

```diff
@@ -8,6 +8,7 @@
  */
 
 import type { Filter } from '@kbn/es-query';
+import type { TimeSlice } from '@kbn/controls-schemas';
```

Comment:
HMMM. How bad would it be to just define `TimeSlice` in `presentation_publishing` instead, do you think? It feels strange to have a controls dependency here 🤔